### ISSUE TITLE
poetry version -s prints new warning to the stdout

### ISSUE THREAD

**Issue by MariuszBielecki288728** [mtkennerly/poetry-dynamic-versioning#101]
Hi,
In the latest release of your package there is an information:
> The plugin now prints a warning if its configuration is invalid. Right now, this just checks for unknown keys.

Not sure if it desirable, but this warning is being printed to stdout and our scripts that uses `poetry version -s` command to pragmatically generate the version string are catching the warning too.
example:
> poetry version -s 2>/dev/null
```
poetry-dynamic-versioning configuration issues:
  - Unknown key: tool.poetry-dynamic-versioning.substitution.enable
22.10.5.dev11
```
It broke some scripts that depended on this version to not contain any additional output, for example:
```
echo -n "SERVICE_VERSION=$(poetry version -s)" > $CI_PROJECT_DIR/version.env
```

What do you think about printing such warning to stderr?

Best regards!

**Comment by mtkennerly** [comment]
Sorry about that. Stderr does make more sense. I'll get this updated.

**Comment by mtkennerly** [comment]
This is fixed now in v0.21.1.
